playwright.config.js

```diff
@@ -72,12 +72,11 @@ export default defineConfig({
     {
       name: 'chromium',
       use: { ...devices['Desktop Chrome'] }
-    }
-
-    // {
-    //   name: 'firefox',
-    //   use: { ...devices['Desktop Firefox'] }
-    // }
+    },
+    {
+      name: 'firefox',
+      use: { ...devices['Desktop Firefox'] }
+    },
```

Comment:
Noice!
Does this mean you'll need to regen a bunch of baseline images and have them potentially be browser specific?

Reply:
Define "env". Like, you mean Stage will be different than Heroku and Prod?